src/components/Footer/Footer.tsx

```diff
@@ -1,18 +1,50 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import { NavigationLink } from '../NavigationLink/NavigationLink';
 
-export const Footer = () => (
-  <nav>
-    <ul>
-      <li>
-        <Link to="#">Github</Link>
-      </li>
-      <li>
-        <Link to="#">Contacts</Link>
-      </li>
-      <li>
-        <Link to="#">Rights</Link>
-      </li>
-    </ul>
-  </nav>
+import './Footer.scss';
+import Logo from '../../img/nice_gadgets_logo.png';
+import BackToTop from '../../img/back_to_top.svg';
+
+export const Footer: React.FC = () => (
+  <footer className="footer">
+    <div className="footer__content">
+      <Link className="footer__logo footer__link" to="index.html">
+        <img src={Logo} alt="Nice Gadgets logo" />
+      </Link>
+
+      <nav className="footer-nav">
+        <ul className="footer-nav__list">
+          <li className="footer-nav__item">
+            <NavigationLink
+              className="footer-nav__link"
+              to="/"
+              title="Gadgets"
```

Comment:
```suggestion
              title="Github"
```
![image](https://github.com/fe-feb23-department-6/phone_store/assets/87998640/82bf06a1-114e-4014-addd-4cec98b57bca)

Seems like it will be better to use just Link, not NavigationLink component